testing/constraints-3.7.txt

```diff
@@ -6,10 +6,10 @@
 # e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
 # Then this file should have foo==1.14.0
 db-dtypes==0.3.1
-google-api-core==1.14.0
-google-auth==1.4.1
+google-api-core==1.21.0
+google-auth==1.18.0
```

Comment:
Doesn't match the minimum constraint in `setup.py`.

Reply:
Updated `setup.py`. Needed this version due to updated `google-api-core` (via `google-cloud-bigquery`)